src/ontology/config/README.md

```diff
@@ -0,0 +1,23 @@
+## Codebook: `src/ontology/config/`
```

Comment:
@nicolevasilevsky @sabrinatoro Hey guys; I added some documentation for these files. Just want you to be aware of where it is for future reference.